helm/etcd-backup-operator/templates/cronjob.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v2beta1
+apiVersion: batch/{{ if ge (.Capabilities.KubeVersion.Minor | int) 21 }}v1{{ else }}v1beta1{{ end }}
```

Comment:
We already have both versions since at least v1.18, AFAIK all installations are v1.20 or am I wrong? What about setting it to v1?
```
batch/v1
batch/v1beta1
```

Reply:
let me try

Reply:
On godsmack, 1.20.5, it does not work

```
unable to recognize "": no matches for kind "CronJob" in version "batch/v1"
```

Reply:
Oh that's suprising to me. I was testing victory and puma